taskcluster/tc-tests-utils.sh

```diff
@@ -306,24 +306,13 @@ assert_correct_warning_upsampling()
   assert_shows_something "$1" "erratic speech recognition"
 }
 
-assert_tensorflow_version()
```

Comment:
This was useful to ensure we don't make mistake with the TensorFlow / DeepSpeech repo dance. And we still expose the TensorFlow version in the native C++ client, so we should keep that.

Reply:
With this PR the TF version is no longer exposed. It's compiled into libdeepspeech.so, but not exposed.

Reply:
Which was your suggestion: https://github.com/mozilla/DeepSpeech/issues/2687#issuecomment-577629637

Reply:
Don't we expose here? https://github.com/mozilla/DeepSpeech/pull/2745/files#diff-5c1b1b3f0263742d08fb2b43f954fed5R265

Reply:
Right, that's *built* into libdeepspeech.so, but not exposed anywhere. It used to be printed to stderr from `DS_PrintVersions()`. I can add a `DS_TensorFlowVersion()` API too if you feel it's useful to keep, I was just going from our discussion in issue #2687.

Reply:
Aaah, I see what you mean, sorry. We make the test load a model and then parse stderr as before.

Reply:
Is there a consistent model file that will be available on all tests? We vary between `${TASKCLUSTER_TMP__DIR}` and `${DATA_TMP_DIR}` for TFLite.

Reply:
> Is there a consistent model file that will be available on all tests? We vary between `${TASKCLUSTER_TMP__DIR}` and `${DATA_TMP_DIR}` for TFLite.

That's only going to be an issue on Android I guess, but I don't think it's a big deal, we don't need a model for `deepspeech --help`, so you should not care ?

Reply:
I just disabled the version test on Android. `deepspeech --help` does not print the TF version.